internal/impl/postgresql/pglogicalstream/sanitize/sanitize.go

```diff
@@ -366,6 +366,18 @@ func SQLQuery(sql string, args ...any) (string, error) {
 // ValidatePostgresIdentifier checks if a string is a valid PostgreSQL identifier
 // This follows PostgreSQL's standard naming rules
 func ValidatePostgresIdentifier(name string) error {
+	if parts := strings.Split(name, "."); len(parts) == 2 {
+		if err := ValidatePostgresIdentifier(parts[0]); err != nil {
+			return fmt.Errorf("invalid schema identifier: %s", err)
+		}
+		name = parts[1]
+	}
+
+	// Strip quotes if they are present
+	if strings.HasPrefix(name, "\"") && strings.HasSuffix(name, "\"") {
+		name = strings.Trim(name, "\"")
```

Comment:
Quoted identifiers don’t need the below verification, they follow different rules (no NULs, any double-quote characters must be doubled).

Reply:
Yes will fixup in a bit